patches/iconv.c.patch

```diff
@@ -0,0 +1,64 @@
+diff --git a/src/iconv.c b/src/iconv.c
+index e3932e5..94ffe9c 100644
+--- a/src/iconv.c
++++ b/src/iconv.c
+@@ -672,9 +672,24 @@ static void conversion_error_other (int errnum, const char* infilename)
+         infilename,line,column);
+ }
+ 
++#ifdef __MVS__
++static int map_encoding_to_ccsid (const char* encoding)
++{
++  /* For now, this is 'short' - just handle 1047 and 
++   * have everything else be ISO8859-1
++   * Use a lookup table to do this 'right'
++   */
++  if (!strcmp(encoding, "IBM-1047") || ! strcmp(encoding, "IBM1047")) {
++    return 1047;
```

Comment:
Should we consider using https://www.ibm.com/docs/en/zos/2.2.0?topic=lf-toccsid-convert-codeset-name-coded-character-set-id ?